msb/rawlogger/msb_rawlogger.py

```diff
@@ -15,15 +15,16 @@ def signal_handler(sig, frame):
 
 
 class RawLoggerService:
-    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber):
+    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber, excluded_topics):
```

Comment:
The excluded topics are already in the config, it is not necessary to pass them in here again.

Reply:
Incorperated comment into the code